lib/scoring.rb

```diff
@@ -1,9 +1,61 @@
+require 'awesome_print'
+require 'pry'
+require_relative 'player'
+
 module Scrabble
   class Scoring
+    attr_reader :word, :letter
+
+    @@letters_hash = { ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"] => 1, ["d", "g"] => 2, ["b", "c", "m", "p"] => 3, ["f", "h", "v", "w", "y"] => 4, ["k"] => 5, ["j", "x"] => 8, ["q", "z"] => 10 }
```

Comment:
Good data structure for this, but it probably doesn't need to be a class variable.  A constant would be a better choice.